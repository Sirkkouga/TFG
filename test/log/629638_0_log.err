INFO - 06/04/24 15:51:21 - 0:00:00 - Read 1071 words from the vocabulary file.

INFO - 06/04/24 15:51:21 - 0:00:00 - 16437 words (1071 unique) in 140 sentences.
INFO - 06/04/24 15:51:21 - 0:00:00 - Job completed successfully
INFO - 06/04/24 15:51:21 - 0:00:00 - Exiting after successful completion
