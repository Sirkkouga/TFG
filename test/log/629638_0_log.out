submitit INFO (2024-06-04 15:51:18,522) - Starting with JobEnvironment(job_id=629638, hostname=veuc09, local_rank=0(1), node=0(1), global_rank=0(1))
submitit INFO (2024-06-04 15:51:18,522) - Loading pickle: /home/usuaris/veu/javier.abella/TFG/test/log/629638_submitted.pkl
Saving the data to /home/usuaris/veu/javier.abella/TFG/test/java.train.cl.0.bpe.pth ...
submitit INFO (2024-06-04 15:51:21,672) - Job completed successfully
submitit INFO (2024-06-04 15:51:21,673) - Exiting after successful completion
